packages/yoastseo/src/languageProcessing/researches/readingTime.js

```diff
@@ -36,24 +40,38 @@ export default function( paper ) {
 		sv: 199,
 		tr: 166,
 	};
-	const minutesPerImage = 0.2;
 
-	let wordsPerMinuteScore = wordsPerMinute[ language ];
+	const charactersPerMinute = {
+		ja: 357,
+	};
+
 
-	// If the language is not on the list, assign the average of all language-dependent reading times as the score.
-	if ( ! wordsPerMinuteScore ) {
+	const wordsPerMinuteScore = wordsPerMinute[ language ];
+	const charactersPerMinuteScore = charactersPerMinute[ language ];
+
+	let wordsOrCharsPerMinuteScore;
+
+	if ( isFeatureEnabled( "JAPANESE_SUPPORT" ) && charactersPerMinuteScore ) {
+		wordsOrCharsPerMinuteScore = charactersPerMinuteScore;
```

Comment:
I think its better to calculate the minutesToReadText in this if-statement directly, I think that would make the code a little bit more readable!

Reply:
I very much like the improved code! I think it's far more readable now. Thanks! 👍